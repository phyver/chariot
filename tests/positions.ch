-- data data

-- tuhesotuhsaoehuoesha

-- data data

(*   utehsuntaoeh *)

-- data data

data nat where Zero : nat
        | Succ : nat -> nat

-- data data

(* utehstuaoehsunhaoeuheaost
  (* stuhaoeunhaoe
     uetasuhseauhtsaoehu
  *)
  utaoesuhtaoesuhao
*)

data data



