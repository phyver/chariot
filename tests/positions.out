--** parse error line 23, column 6 (token <data>)
